app/mailers/faculty_notifications_mailer.rb

```diff
@@ -19,4 +19,12 @@ def scholarsphere_deposit_failure(user, deposit)
          from: 'scholarsphere@psu.edu',
          reply_to: 'scholarsphere@psu.edu'
   end
+
+  def wrong_file_version(publications)
+    @publications = publications
+    mail to: "#{User.find(@publications.first.activity_insight_oa_files.first.user_id).webaccess_id}@psu.edu",
```

Comment:
I think that this line could be refactored by using the new Publication#activity_insight_file_uploader method.

It could possibly be refactored further by wrapping the user object in an instance of the UserProfile class which already has a method for assembling the email address for a user.



Something like
```ruby
profile = UserProfile.new(@publications.first.activity_insight_file_uploader)
mail to: profile.email
```

That's not really any more concise, but it eliminates a bunch of duplication and method chaining, and it improves readability - all of which is far more important.